apps/crn-server/src/data-providers/contentful/reminder.data-provider.ts

```diff
@@ -879,6 +879,15 @@ const getPublishedResearchOutputVersionRemindersFromQuery = (
     return [];
   }
 
+  items.sort((reminderA, reminderB) => {
```

Comment:
we are already doing the exact same sort here: https://github.com/yldio/asap-hub/blob/f9cffb6533da1de710d124646a0387ffb5762d3e/apps/crn-server/src/data-providers/contentful/reminder.data-provider.ts#L193-L198
any reason why this needs to occur again earlier?

Reply:
The earlier sort is so that if multiple versions have been created for one output within the time limit we return a reminder for the latest only. I sort the output versions and then when a reminder is generated for an output version keep track of its related output. Subsequent versions with the same related output will be ignored.

The later sort sorts all reminders (eg: output publish, output version publish, events etc).